tests/tests/kernel/datatypes/ezimage/ezimagealiashandler_regression.php

```diff
@@ -12,12 +12,6 @@ class eZImageAliasHandlerRegression extends ezpDatabaseTestCase
 {
     protected $backupGlobals = false;
 
-    public function __construct()
-    {
-        parent::__construct();
-        $this->setName( "eZImageAliasHandler Regression Tests" );
-    }
-
```

Comment:
The constructor what breaking the `@dataProvider`. Is setting the name really usefull.